app/src/renderer/components/common/ShortBech32.vue

```diff
@@ -1,7 +1,7 @@
 <template lang="pug">
 .short-bech32
   .address(@click.prevent.stop="copy" v-tooltip.top="address")#address {{ shortBech32 }}
-  .success(:class="{active:showSuccess}")
+  .copied(:class="{active:showSuccess}")
```

Comment:
my comment wasn't about the class but about the DOM element being there all the time. did you check if this has sth. todo with the animation?

Reply:
ah i misunderstood. still, i think this is better. 

it _does_ have to do with the animation. currently, the element is `position: relative` and so if you use a `v-if` then it will bump things around it. also, transitions work when changing css properties, not inserting new elements into the DOM.

it could be changed to allow for a `v-if` but i'm not sure that would be _better_.

Reply:
hm ok